reservations/tests/factories.py

```diff
@@ -14,6 +14,11 @@ class Meta:
         model = "reservations.ReservationCancelReason"
 
 
+class ReservationDenyReasonFactory(DjangoModelFactory):
+    class Meta:
+        model = "reservations.ReservationCancelReason"
```

Comment:
Should be `ReservationDenyReason`.

Reply:
Good catch. I actually fixed this one in the draft PR but it should be here :woozy_face: